docs/aerodromes/Bankstown.md

```diff
@@ -87,8 +85,8 @@ Helicopters should track inbound at `A007` via one of the normal inbound points
     Helicopters tracking via CSTH will pass over the runway complex midfield at `A005` to join downwind.  Be mindful of aircraft in the fixed-wing circuit and pass traffic information to both aircraft prior to the fixed-wing aircraft turning final.  
 
     Example:  
-    *"LOI, traffic is a helicopter overflying the aerodrome to the north at 500ft, runway left, cleared touch and go"*  
-    *"YZD, traffic is a Cherokee turning final for runway left, overfly midfield at 500ft, join downwind main pad"*
+    *"LOI, traffic is a helicopter overflying the aerodrome to the north at A005, runway left, cleared touch and go"*  
+    *"YZD, traffic is a Cherokee turning final for runway left, overfly midfield at A005, join downwind main pad"*
```

Comment:
being picky, but are we doing A005 for example phraseology too?

Reply:
i reckon so